gammapy/maps/axes.py

```diff
@@ -2044,6 +2059,31 @@ def assert_names(self, required_names):
         except ValueError:
             raise ValueError(message)
 
+
+    def rename(self, names, new_names):
+        """Rename the axes.
+    
+        Parameters
+        ----------
+        names : list or str
+            Names of the axes
+        new_names : list or str
+            New names of the axes (list must be of same length than `names`).
+        
+        Returns
+        -------
+        axes : `MapAxes`
+            Renamed Map axes object
+        """
+        axes = self.copy()
+        if isinstance(names, str):
+            names = [names]
+        if isinstance(new_names, str):
+            new_names = [new_names]
+        for name, new_name in zip(names, new_names):
+            axes[name]._name = new_name
```

Comment:
This does not really make a copy of the `MapAxis`, just use the `MapAxis.rename()` method here you introduced earlier.

Reply:
but here axes is self.copy() which already copied each axis